depends/packages/native_cctools.mk

```diff
@@ -18,15 +17,9 @@ endef
 ifneq ($(strip $(FORCE_USE_SYSTEM_CLANG)),)
 define $(package)_preprocess_cmds
   mkdir -p $($(package)_staging_prefix_dir)/lib && \
-  cp $(llvm_lib_dir)/libLTO.so $($(package)_staging_prefix_dir)/lib/ && \
-  cp -f $(BASEDIR)/config.guess $(BASEDIR)/config.sub cctools && \
```

Comment:
Why this line removed? I understand that it works now without it, but:
- there were related problems in the past, for example https://github.com/bitcoin/bitcoin/pull/22526
- this change is unrelated to the PR topic and omitted in the PR description

Reply:
The new source has a newer `config.guess` and `config.sub`, which has the arm triple we need.

> there were related problems in the past,

Ok? However the triple isn't going to be removed in the future, so I'm not sure what you're point is.

> this change is unrelated to the PR topic and omitted in the PR description

How is it unrelated? Now that we are using newer sources, we don't need to copy over additional files.

Reply:
Also, all of this code is going to be deleted in the near future in any case (#21778). The only reason it's being updated now is to avoid an awkward split in the GCC toolchains used in the Guix builds, otherwise, we'd avoid bumping it again.